extensions/amp-lightbox-gallery/0.1/service/lightbox-manager-impl.js

```diff
@@ -188,6 +188,10 @@ export class LightboxManager {
                 && slide.getAttribute('lightbox') !== lightboxGroupId);
         if (!shouldExcludeSlide) {
           slide.setAttribute('lightbox', lightboxGroupId);
+          if (this.seen_.includes(slide)) {
```

Comment:
Do you still need to set the attribute if the slide is in `this.seen_`? Or will it already be set?

Reply:
That's true. Yes, it will have already been set if it's in seen.